### ISSUE TITLE
Scope accessibility (PI) test plan

### ISSUE THREAD

**Issue by m8ttyB** [mozilla-lockbox/lockbox-ios#99]
The intent is to generate a v0.1 of an accessibility test plan specific to iOS. This is meant to be comprehensive enough that we can discuss the scope of the testing and what is needed to accomplish it. This is not meant to detail individual test cases, although I suspect that during this spike we'll identify some as we're scoping the work.

- [x] Partner with yzen (possibly njpark), capture testing considerations as well as tooling requirements and document them.
  - UI and design
  - voiceover

- [x] Integrate feedback as well as get sign-off from engineering and the product team.

## Acceptance criteria

- [x] The final product, the test plan, should be added to the [iOS docs in github](https://github.com/mozilla-lockbox/lockbox-ios/tree/master/docs).

**Comment by devinreams** [comment]
@mozilla-lockbox/ux have any accessibility reviews of the app designs occurred upstream? I ask so I know how much help we'd like to ask for ("never looked at, already implemented" vs "already reviewed, just need double check of what's here plus new last-minute additions").

**Comment by changecourse** [comment]
@devinreams we've done our due diligence throughout the design process to ensure adequate color contrast on the designs across the UI, but no official accessibility review has occurred on our end. 

#317 is the only UI accessibility issue we've caught on the current build thus far

Voiceover testing throughout would be the big potential/outstanding need for accessibility if it isn't accounted for as of yet.
